web_generator/bin/update_idl_bindings.dart

```diff
@@ -67,11 +67,13 @@ $_usage''');
 
   // Run app with `node`.
   final generateAll = argResult['generate-all'] as bool;
+  final inputFiles = argResult.rest;
   await runProc(
     'node',
     [
       'main.mjs',
       '--idl',
+      for (String inputFile in inputFiles) '--input=$inputFile',
       '--output=${p.join(_webPackagePath, 'lib', 'src')}',
```

Comment:
If `inputFiles` is non-empty, should we require users specify an output so we don't place the files in the `package:web` destination?